grpcdogfood/client/dogfood_client.go

```diff
@@ -0,0 +1,85 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	pb "github.com/DataDog/chaos-controller/grpcdogfood/chaosdogfood"
+	"google.golang.org/grpc"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
+)
+
+const serverAddr = "localhost:50051"
```

Comment:
I assume this client & server will eventually be run in kubernetes? **Optional:** Should we make the serverAddr a go flag in this PR for both server & client?

```
var serverAddr string

func init() {
	flag.StringVar(&serverAddr, "server-addr", ":50051", "Address to listen on")
}

func main() {
 	....
}
```

Reply:
Just checked out the https://pkg.go.dev/flag go package and it looks perfect! Thanks for the recommendation. My next PR I'm working on now allows us to deploy the service and client to K8s (wanted to split that off from business logic / basic app), so I will implement this flag in that PR. 🥇